Graphics/GraphicsEngineOpenGL/src/SamplerGLImpl.cpp

```diff
@@ -102,7 +102,15 @@ SamplerGLImpl::SamplerGLImpl(IReferenceCounters* pRefCounters, class RenderDevic
     }
 
     if (SamPrpos.AnisotropicFilteringSupported) // Can be unsupported
+    {
+#if defined(DILIGENT_DEBUG) || defined(DILIGENT_DEVELOPMENT)
+        float maxLevel;
+        glGetFloatv(GL_MAX_TEXTURE_MAX_ANISOTROPY_EXT, &maxLevel);
```

Comment:
This will produce error on all platforms that don't support the `GL_MAX_TEXTURE_MAX_ANISOTROPY_EXT` extension